packages/common/src/api/user.ts

```diff
@@ -147,17 +145,15 @@ const userApi = createApi({
           type,
           method
         }: Pick<GetUSDCTransactionListArgs, 'userId' | 'type' | 'method'>,
-        { audiusSdk, audiusBackend }
+        { audiusSdk }
       ) => {
-        const { data: encodedDataMessage, signature: encodedDataSignature } =
-          await audiusBackend.signDiscoveryNodeRequest()
         const sdk = await audiusSdk()
         const { data } = await sdk.full.users.getUSDCTransactionCount({
           id: Id.parse(userId!),
           type,
           method,
-          encodedDataMessage,
-          encodedDataSignature
+          encodedDataMessage: '', // TODO: remove, handled by sdk
+          encodedDataSignature: '' // TODO: remove, handled by sdk
```

Comment:
Do we have a ticket for this? Or should we just do it now? 👀

Reply:
I was going to open a PR for this right after -- I just don't want to make this one so large, since it will require discovery changes too

Reply:
made ticket